app/controllers/api/api_controller.rb

```diff
@@ -14,7 +14,6 @@ class ApiController < ApplicationController
     rescue_from Api::UnauthorizedTokenError,             with: :not_authenticated
     rescue_from Api::UnsupportedMediaType,               with: :unsupported_media_type
     rescue_from ControlControl::AccessDenied,            with: :not_authorized
-    rescue_from ActiveRecord::StatementInvalid,          with: :bad_query
```

Comment:
I assume Json Schema Validator is checking id formats now so we don't need to rescue this one explictly?